### PR TITLE
More translations

### PR DESCRIPTION
**Pull request by zeuner**
I'm currently filling in the missing translations as of the latest minetest game. It may be best to wait with the merge until they are all done. So, this is also to allow comments already while the work is in progress.

### CODE REVIEW

File: DE.lua [hunk 1]
```diff
@@ -307,41 +307,69 @@ return {
...
+	{"stairs:stair_ice",				"Eistreppe"},
+	{"stairs:stair_snowblock",			"Schneeblocktreppe"},
 	{"stairs:stair_wood",				"Holztreppe"},
 	{"stairs:stair_junglewood",			"Dschungelholztreppe"},
 	{"stairs:stair_pinewood",			"Kieferholztreppe"},
 	{"stairs:stair_acacia_wood",			"Akazienholztreppe"},
+	{"stairs:stair_aspen_wood",			"Espenholztreppe"},
 	{"stairs:stair_stone",				"Steintreppe"},
+	{"stairs:stair_stone_block",			"Steinblocktreppe"},
+	{"stairs:stair_mossycobble",			"bemoste Pflastersteintreppe"},
```
Comment (HybridDog) [inline DE.lua]: bemooste

File: DE.lua [hunk 2]
```diff
@@ -377,6 +405,11 @@ return {
 	{"vessels:glass_fragments",			"Glasscherben"},
 	{"vessels:shelf",				"Gefäßschrank"},
 
+	-- walls
+	{"walls:cobble",				"Pflastersteinmauer"},
+	{"walls:mossycobble",				"bemoste Pflastersteinmauer"},
```
Comment (HybridDog) [inline DE.lua]: bemooste

**Comment by MuhdNurHidayat** [comment]
I'm waiting for the missing source translations from newer minetest game to be filled, and gets merge first, then I'll start translating to my language.

**Comment by zeuner** [comment]
Great!

original message follows:

**Comment by zeuner** [comment]
Great!

original message follows:

**Comment by zeuner** [comment]
Thanks.

original message follows:

**Pull request merged**